### ISSUE TITLE
Last tpch benchmarks don't show statistics and cluster url is missing

### ISSUE THREAD

**Issue by phofl** [coiled/benchmarks#1163]
Scrolling down here: https://benchmarks.coiled.io/details/Python%203.9-tpch-test_dask-test_query_3.html

The stats are missing, the ci run says the query finished but that's impossible in 12 seconds, we are not that fast...

**Comment by ntabris** [comment]
In case it's helpful, here's link to last cluster there:

https://cloud.coiled.io/clusters/301559/information

From what I see there, the query 3 was ~12 seconds:

https://cloud.coiled.io/clusters/301559/information?computation=&sinceMs=1698413793609&untilMs=1698413813609

**Comment by phofl** [comment]
Yeah this is probably because of some meta gathering thing that's not working very well at the moment. The actual query should take way longer than that

**Comment by phofl** [comment]
General note for others: You can get to the cluster via the grafana dashboard

**Comment by mrocklin** [comment]
Currently we're recording using the `benchmark_time` fixture (should have timings), but not `benchmark_all` (would have memory, cpu, etc.. recorded as well).  I can find a way to add back in the previous fixtures if folks would find those useful.  (this isn't exactly the problem you're referring to here, but I suspect that it's similar)
